modules/core/client/views/core.client.views.sidenav.left.html

```diff
@@ -0,0 +1,14 @@
+<md-sidenav class="md-sidenav-left md-whiteframe-z1" md-component-id="left">
+  <md-toolbar class="md-theme-blue-grey md-hue-3">
+    <h1 class="md-toolbar-tools">
+      <ng-md-icon icon="apps"></ng-md-icon>
```

Comment:
What is this icon here for?  It doesn't do anything.  If you want it for something thats fine, but we need to add a close icon that executes vm.close()